src/transformers/models/gptj/modeling_gptj.py

```diff
@@ -904,7 +904,7 @@ def __init__(self, config):
 
     @add_start_docstrings_to_model_forward(GPTJ_INPUTS_DOCSTRING.format("batch_size, sequence_length"))
     @add_code_sample_docstrings(
-        checkpoint=_CHECKPOINT_FOR_DOC,
+        checkpoint="ydshieh/tiny-random-gptj-for-sequence-classification",
```

Comment:
The one `hf-internal-testing/tiny-random-gptj` has some shape issue, see my comment in PR description.